renderdoc/driver/vulkan/wrappers/vk_shader_funcs.cpp

```diff
@@ -726,10 +715,6 @@ bool WrappedVulkan::Serialise_vkCreateGraphicsPipelines(
     VkPipeline pipe = VK_NULL_HANDLE;
 
     VkRenderPass origRP = CreateInfo.renderPass;
-    VkPipelineCache origCache = pipelineCache;
-
-    // don't use pipeline caches on replay
-    pipelineCache = VK_NULL_HANDLE;
```

Comment:
Could you actually add a chicken bit here with an `RDOC_CONFIG`? Just in case this causes any issues in the wild to let us go back to turning off caches on replay.